devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -232,6 +236,21 @@ def getConfigurationCommands() {
 		// if there's nothing to configure, we're configured
 		state.configured = true
 	}
+
+	if (isEverspring()) {
+		if (!state.alarmLength) {
+            state.alarmLength = everspringDefaultAlarmLength
+        }
```

Comment:
your spacing is still mixed up.

Your editor should have a way to convert spaces->tabs for you

Reply:
Ok, I've converted that and few more lines into tabs in my IDE.